scripts/automation/Radius/readme.md

```diff
@@ -75,106 +84,182 @@ After selecting the User Group, view the url for the user group it should look s
+![select network](./images/mac_radiusNetwork_step1.png)
 
-1. Extract the private key: `openssl pkcs12 -in certfile.pfx -nocerts -out /projectDirectory/Radius/Cert/radius_ca_key.pem`
-   * Replace certfile.pfx to the file path of your .pfx file. Make sure the `radius_ca_key.pem` is saved or moved to `/projectDirectory/Radius/Cert/` directory
-   * This command will prompt for the .pfx password. NOTE: Please DO NOT FORGET the password as you will need it when generating user certificates.
-2. Extract the certificate: `openssl pkcs12 -in certfile.pfx -nokeys -out /projectDirectory/Radius/Cert/radius_ca_cert.pem`
-   *  Replace certfile.pfx to the file path of your .pfx file. Make sure the  `radius_ca_cert.pem` is saved or moved to `/projectDirectory/Radius/Cert/`  directory
-   *  Again, this command will prompt for the .pfx password
+The user is prompted to validate the certificate, click "Continue"
 
-NOTE: You will get prompted with the Key password Generating user certificates. The password will be saved as an Environment Variable.
+![select network](./images/mac_radiusNetwork_step2.png)
 
-### MacOS
+After validating the certificate, the user is prompted to enter their password to assign the Radius SSID to the selected certificate:
 
-In MacOS a user simply needs to select the radius network from the wireless networks dialog prompt. An option to authenticate with a certificate should be presented. After selecting the user's certificate, the MacOS user is prompted to enter their system password. They can do this and select "Always Allow" or will otherwise be prompted to enter their password three times.
+![select network](./images/mac_radiusNetwork_step3.png)
+
+After entering the password, the user as asked again to sign using this certificate, enter the password and select "Always Allow"
+
+![select network](./images/mac_radiusNetwork_step4.png)
+
+After entering the password, the user as asked again to access using this certificate, enter the password and select "Always Allow"
+
+![select network](./images/mac_radiusNetwork_step5.png)
+
+The user should then be connected to the radius network.
 
 ### Windows
 
-In Windows, select the radius network from the wireless networks dialog prompt, an option to select a certificate should be displayed. Select the certificate which corresponds with the user on the device. Select "OK"
+In Windows, select the radius network from the wireless networks dialog prompt, an option to select a certificate should be displayed. Select the certificate which corresponds with the user on the device. Select "OK":
 
 ![select network](./images/windows_select_network.png)
 
 Before Connecting, users can view the authentication source. Click "Connect" to connect to the network, no password is necessary.
 
 ![select network](./images/windows_auth.png)
 
+The user should then be connected to the radius network.
+
 ### Troubleshooting
 
 #### Clearing Commands Queue
```

Comment:
I think we should notate more here saying that this will clear out ALL of their queued commands, not just the ones pertaining to the Radius Certs

Reply:
Cool Agreed, changing queue function